README.md

```diff
@@ -1,2 +1,37 @@
-# fantasy-land
-Specification for interoperability of common algebraic structures in PHP
+# FantasyLand [![Build Status](https://travis-ci.org/functional-php/fantasy-land.svg?branch=master)](https://travis-ci.org/functional-php/fantasy-land)
+## Introduction
+
+Specification for interoperability of common algebraic structures in PHP.
```

Comment:
The sentence from Fantasy Land seems better to me :

"This project specifies interoperability of common algebraic structures in PHP."